container_test.go

```diff
@@ -438,20 +438,6 @@ func TestProvide(t *testing.T) {
 	require.NotNil(t, p1.c1.gc1, "Grandchild1 must have been registered")
 }
 
-func TestConcurrentAccess(t *testing.T) {
```

Comment:
We should do this in the PR where we remove locks, otherwise it hard to track commits..
e.g. you are still using locks in `ValidateReturnTypes`